### ISSUE TITLE
Interface of overridden method 'START' of base 'FB_ENGINE' doesn't match declaration

### ISSUE THREAD

**Issue by loraul** [Aliazzzz/OOP-Concept-Examples-in-CODESYS-V3#1]
Hi Aliazzzz,
    I think there is a little mistake in OPP examples. When I try to compile the examples I get this error:

_Start [FB_NewEngine]: C0094:  Interface of overridden method 'START' of base 'FB_ENGINE' doesn't match declaration_
   
Probably, the problem is because the declaration of variable inputs in method START should be the same in the base and child class. Now the declarations are different:
 
**BASE CLASS:**
_VAR_INPUT
   rVelocity : REAL;
   iGear     : INT;
END_VAR_

**CHILD CLASS:**
_VAR_INPUT
   fVelocity : REAL;
   nGear     : INT;
END_VAR_

Thanks for your job.

**Comment by jclavier44** [comment]
Yep found the same issue, child class must have at least "rVelocity" and "iGear" like parent method
